pkg/server/plugin/keymanager/v1.go

```diff
@@ -94,6 +99,28 @@ func (v1 V1) makeKey(id string, pb *keymanagerv1.PublicKey) (Key, error) {
 	}, nil
 }
 
+func (v1 *V1) convertKeyType(t KeyType) (keymanagerv1.KeyType, error) {
+	switch t {
+	case KeyTypeUnset:
+		return keymanagerv1.KeyType_UNSPECIFIED_KEY_TYPE, v1.Error(codes.InvalidArgument, "key type is required")
+	case ECP256:
+		return keymanagerv1.KeyType_EC_P256, nil
+	case ECP384:
+		return keymanagerv1.KeyType_EC_P384, nil
+	case RSA2048:
+		return keymanagerv1.KeyType_RSA_2048, nil
+	case RSA4096:
+		return keymanagerv1.KeyType_RSA_4096, nil
+	default:
+		return keymanagerv1.KeyType_UNSPECIFIED_KEY_TYPE, v1.Errorf(codes.Internal, "facade does not support key type %s", t)
```

Comment:
nit: we use %q for this in other places, which I prefer
```suggestion
		return keymanagerv1.KeyType_UNSPECIFIED_KEY_TYPE, v1.Errorf(codes.Internal, "facade does not support key type %q", t)
```